conditions/README.md

```diff
@@ -0,0 +1,341 @@
+| extension[index].extension[0].valueCode                | either `"child"` if `LinkSet.component.statementRef.id [@root]` references another Link Set or `"parent"` if the mapped Link Set is referenced by another Link Set in the same manner <sup>3</sup> |
+| extension[index].extension[1].url                      | fixed value = `"target"` <sup>3</sup>                                                                                                                                                              |
+| extension[index].extension[1].valueReference.reference | Reference to the child / parent Condition  <sup>3</sup>                                                                                                                                            |
+| identifier[0].system                                   | "https://PSSAdaptor/{{odsCode}}}" - where {{osdCode}} is the ODS code of the losing practice                                                                                                       |
+| identifier[0].value                                    | `LinkSet / id [@root]`                                                                                                                                                                             |
+| clinicalStatus                                         | If `LinkSet / code [@code]` equals `394775005` then `inactive` otherwise `active`                                                                                                                  |
+| category[0].coding[0].system                           | fixed value = "https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-ConditionCategory-1"                                                                                                            |
+| category[0].coding[0].code                             | fixed value = `problem-list-item`                                                                                                                                                                  |
+| category[0].coding[0].display                          | fixed value = `Problem List Item`                                                                                                                                                                  |
+| code                                                   | mapped from `ObservationSatement / code` where the `Observationstatement` is referenced in `LinkSet / component / statementRef / id [@root]`                                                       |
```

Comment:
`ObservationStatement` appears to be coming from `LinkSet / conditionNamed / namedStatementRef / id [@root]` or am I looking at the wrong thing 😬.

Line 190 of ConditionMapper.java

Reply:
your right, it's `namedStatementRef`, thanks